lib/options.js

```diff
@@ -28,7 +28,10 @@ function defaultOptions() {
             host: process.env.DATABASE_HOST,
             port: process.env.DATABASE_PORT || 5432,
             database: process.env.DATABASE_NAME || 'unleash',
-            ssl: process.env.DATABASE_SSL,
+            ssl:
+                process.env.DATABASE_SSL != null
+                    ? JSON.parse(process.env.DATABASE_SSL)
+                    : {},
```

Comment:
I would prefer the default to be `false`. Passing in an empty object will still enable SSL with default options (e.g. not allowing self-signed certificates, etc).